client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/users/UserScreen.kt

```diff
@@ -455,8 +455,8 @@ class UserScreen :
                     BpDropdownMenuItem(
                         onClick = {
                             when (it.text) {
-                                Resources.Strings.edit -> onEditUserMenuItemClicked(user)
-                                Resources.Strings.delete -> onDeleteUserMenuItemClicked(user)
+                                "Edit" -> onEditUserMenuItemClicked(user)
```

Comment:
string! why?

Reply:
I will discuss about it in daily meeting